libs/api/ebpf_api.cpp

```diff
@@ -1165,7 +1165,10 @@ ebpf_map_get_next_key(fd_t map_fd, _In_opt_ const void* previous_key, _Out_ void
     uint32_t type;
     ebpf_handle_t map_handle = ebpf_handle_invalid;
 
-    ebpf_assert(next_key);
+    if (next_key == NULL) {
```

Comment:
Change `_Out_` to `_Out_opt_` in line 1154 to indicate to the linter that passing NULL should not be an error but is handled by the function.

Reply:
Thanks, fixed.